components/brave_ads/core/internal/ads_impl.h

```diff
@@ -13,6 +13,7 @@
 #include "base/memory/weak_ptr.h"
 #include "brave/components/brave_ads/browser/ads_service_callback.h"
 #include "brave/components/brave_ads/core/internal/account/tokens/token_generator_interface.h"
+#include "brave/components/brave_ads/core/internal/database/database_maintenance.h"
```

Comment:
with the `std::unique_ptr<database::Maintenance>` it is enough to have forward declaration of database::Maintenance.

Reply:
Fixed. And will follow up with transitioning database maintenance to AdsCore.